process/accounts/altered_accounts_processor.go

```diff
@@ -0,0 +1,48 @@
+package accounts
+
+import (
+	"github.com/ElrondNetwork/covalent-indexer-go/api"
+	"github.com/ElrondNetwork/covalent-indexer-go/process/utility"
+	"github.com/ElrondNetwork/covalent-indexer-go/schemaV2"
+)
+
+type alteredAccountsProcessor struct {
+}
+
+// NewAlteredAccountsProcessor creates a new instance of altered accounts processor
+func NewAlteredAccountsProcessor() *alteredAccountsProcessor {
+	return &alteredAccountsProcessor{}
+}
+
+// ProcessAccounts converts accounts data to a specific structure defined by avro schema
+func (ap *alteredAccountsProcessor) ProcessAccounts(apiAlteredAccounts []*api.AlteredAccount) ([]*schemaV2.AccountBalanceUpdate, error) {
```

Comment:
``ConvertAccounts

Reply:
Hmm... good point.
Although, each processor in this repo is called `Processor`, so I would keep it for consistency